MapboxAndroidDemo/build.gradle

```diff
@@ -46,8 +46,8 @@ android {
     productFlavors {
         global {
             isGlobal = true
-            resValue "string", "firebase_app_id", "1:557332920931:android:bdf11618ae45dd61"
-            resValue "string", "firebase_api_key", "AIzaSyCGCYLyLoi31Hv-pkBO0D51zCyxjngJbz8"
+            resValue "string", "firebase_app_id", System.getenv('FIREBASE_APP_ID') ? System.getenv('FIREBASE_APP_ID') : "blank_place_holder_string"
```

Comment:
What happens if you just have `System.getenv('FIREBASE_APP_ID')`? (without the ternary expression / default value)

Reply:
`Build Config field cannot have a null parameter`

<img width="1088" alt="Screen Shot 2019-06-06 at 6 28 16 AM" src="https://user-images.githubusercontent.com/4394910/59036525-518e6f00-8824-11e9-8673-0cde60db51c2.png">